tools/get_release_contributors.py

```diff
@@ -31,26 +36,28 @@ class UnexpectedGithubStatus(RuntimeError):
     """
     Exception which is raised when an API request fails.
     """
-    pass
 
 
 # List of contributor names which should be ignored (such as bots):
-ignore_list = ['@github-actions[bot]', '@imgbot[bot]', '@actions-bot', '@actions-user', '@ImgBotApp', '@dependabot[bot]', '@weblate']
+ignore_list = ['@github-actions[bot]', '@imgbot[bot]', '@actions-bot',
+               '@actions-user', '@ImgBotApp', '@dependabot[bot]', '@weblate']
```

Comment:
Unrelated, but I think the imgbot[bot] can be soon removed.

Reply:
OK, I have no idea

Reply:
Yes, that's just a comment for the future.